presto-orc/src/main/java/com/facebook/presto/orc/writer/DictionaryColumnWriter.java

```diff
@@ -468,6 +468,10 @@ public ColumnStatistics getColumnStatistics()
         {
             return columnStatistics;
         }
+
+        public long getRetainedSizeInBytes() {
+            return sizeOf(dictionaryIndexes) + columnStatistics.getRetainedSizeInBytes();
```

Comment:
nit: I would recommend also including the instance size of the `DictionaryRowGroup` class as well while you are at it. You can see an example here: 

https://github.com/prestodb/presto/blob/master/presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java#L34
https://github.com/prestodb/presto/blob/master/presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java#L156